pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -133,17 +147,36 @@ func (r *ReconcileComputeNodeOpenStack) Reconcile(request reconcile.Request) (re
 		return reconcile.Result{}, err
 	}
 
+	// ScriptsConfigMap
+	err = createOrUpdateComputeNodeOpenstackScriptsConfigMap(r, instance, strings.ToLower(instance.Kind)+"-scripts")
```

Comment:
perhaps we can renamed to "ensureComputeNodeOpenStackScriptsConfigMap

Reply:
done